test/integration/consul-container/test/README.md

```diff
@@ -0,0 +1,123 @@
+# Upgrade Integration Tests
+
+The upgrade integration test is responsible for ensuring that users are able to upgrade between consul versions without experiencing difficulties. The upgrade tests does the following:
+- Ensure that customers are able to upgrade to a new Consul version, and new and existing features work as expected. 
+- This tests aims to ensure a high degree of interoperability between versions of Consul by supporting `n-2` and `n-1` upgrades, ensuring that upgrading to newer versions does not degrade or break functionalities.
+    - Validate users are able to perform a direct upgrade of Consul. For example upgrading from `1.13 → 1.14`.  
+    - Validate users are able to perform skip-version upgrade of Consul. For example upgrading from `1.13 → 1.15`.   
+
+- The upgrade tests also aims to highlight errors that may occur as users attempt to upgrade their current version to a newer version. 
```

Comment:
Seems like there are some redundant lines in this section. Also, may want to highlight the goals. Something like:

```suggestion
The goal of upgrade tests is to ensure problem-free upgrades on supported upgrade paths. At any given time, Consul supports the latest minor release, and two older minor releases, e.g. 1.15, 1.14, and 1.13. Upgrades to any higher version are permitted, including skipping a minor version e.g. from 1.13 to 1.15.
```